annotation/tests/test_jobs.py

```diff
@@ -343,4 +328,203 @@ def test_get_job_attributes_for_post_attribute_not_found():
     mock_session = MagicMock()
     mock_session.query().filter_by().first.return_value = None
     with pytest.raises(FieldConstraintError):
-        get_job_attributes_for_post(mock_session, job_id, "test", (Job.name,))
+        annotation.jobs.services.get_job_attributes_for_post(
+            mock_session, job_id, "test", (Job.name,)
+        )
+
+
+def test_recalculate_file_pages(files: Tuple[File]):
+    mock_session = MagicMock()
+    mock_result = MagicMock()
+    mock_session.query().filter.return_value = mock_result
+    expected_validating_pages = [5, 7]
+    expected_annotating_pages = [3, 5, 6, 7]
+
+    filter_side_effect = (
+        lambda *args: MagicMock(
```

Comment:
you dont use this args
```suggestion
        lambda *_: MagicMock(
```

Reply:
but i do use it in line 347

Reply:
Currently it is line 339